src/rules/comment-word-blacklist/README.md

```diff
@@ -0,0 +1,39 @@
+# comment-word-blacklist
+
+Specify a blacklist of disallowed comments.
+
+```css
+    /* comment */
+/**    ↑
+ * These comments */
+```
+
+## Options
+
+`array`: `"["array", "of", "unprefixed", "functions"]`
+
+Given:
+
+```js
+["/todo/", "badword"]
```

Comment:
Can we swap `/todo/` for `/^TODO:/` please? And update the examples below accordingly?